code/modules/paperwork/paper.dm

```diff
@@ -352,6 +352,16 @@
 	update_icon()
 
 /*
+ * Construction paper
+ */
+
+/obj/item/weapon/paper/construction
+
+/obj/item/weapon/paper/construction/New()
+	..()
+	color = pick("FF0000", "#33cc33", "#ffb366", "#551A8B", "#ff80d5", "#4d94ff")
```

Comment:
it's not random, it specifically chooses among cult colors, otherwise people will metagame like they did with random rune colors